### ISSUE TITLE
Nothing being reported

### ISSUE THREAD

**Issue by josiahsprague** [ryanbagwell/wordpress-sentry#10]
I followed the instructions in readme.txt, installed the plugin and added the DSN from my project in Sentry, but I'm not seeing anything reported. I manually called a function that doesn't exist on 404.php in my theme to try to trigger an error, but still nothing is being reported. Any ideas? Am I missing anything?

**Comment by josiahsprague** [comment]
I have the error reporting level set to `E_ALL`.

**Comment by josiahsprague** [comment]
I am seeing these errors in my PHP error logs, which are not in the standard location, not sure if that matters;
```
[22-Sep-2015 19:14:28 UTC] PHP Fatal error:  Call to undefined function nonexistentfunction() in /path/wp-content/themes/themename/404.php on line 2
[22-Sep-2015 19:35:47 UTC] PHP Parse error:  syntax error, unexpected '@' in /path/wp-content/themes/themename/404.php on line 4
[22-Sep-2015 19:35:57 UTC] PHP Parse error:  syntax error, unexpected '@' in /path/wp-content/themes/themename/404.php on line 4
```

**Comment by josiahsprague** [comment]
I was finally able to get an error to be reported in sentry with;
```
throw new Exception('Test exception.');
```
But I'd like to get PHP Fatal errors, Parse errors, etc to be reported as well. I figured `E_ALL` would report those.

**Comment by v-bartusevicius** [comment]
Fatal errors requires special `shutdown_function` to be registered.
Added it in this PR: https://github.com/ryanbagwell/wordpress-sentry/pull/11

**Comment by ryanbagwell** [comment]
Fixed with 13f2133
